tests/lib/rules/utils/ast-utils.js

```diff
@@ -212,7 +212,6 @@ describe("ast-utils", () => {
                 "//xxglobal foo",
                 "//xxglobals foo",
                 "//xxexported foo",
-                ""
```

Comment:
just removed this line, as it seems useless. (Not sure why I added it 🤣)

Reply:
Removing this line introduced a linting error.

Reply:
it was fixed in a following commit.